### PR TITLE
Remove aliases reference from autoload

### PR DESCRIPTION
**Pull request by dingo-d**
I was trying to set up a test for the `develop` branch so that I can do a step by step instructions on the make blog and there was an error coming from the `autoload.php`.

### Steps to reproduce

1. Go to theme folder and run 

```sh
composer require --dev wptrt/wpthemereview:dev-develop dealerdirect/phpcodesniffer-composer-installer:^0.4.4
```
This installs the latest version:

```sh
./composer.json has been created
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 7 installs, 0 updates, 0 removals
  - Installing squizlabs/php_codesniffer (3.4.0): Downloading (100%)
  - Installing dealerdirect/phpcodesniffer-composer-installer (v0.4.4): Downloading (100%)
  - Installing wp-coding-standards/wpcs (2.0.0): Downloading (100%)
  - Installing phpcompatibility/php-compatibility (9.1.1): Downloading (100%)
  - Installing phpcompatibility/phpcompatibility-paragonie (1.0.1): Downloading (100%)
  - Installing phpcompatibility/phpcompatibility-wp (2.0.0): Downloading (100%)
  - Installing wptrt/wpthemereview (dev-develop 895ed6d): Cloning 895ed6d2ed from cache
dealerdirect/phpcodesniffer-composer-installer suggests installing dealerdirect/qa-tools (All the PHP QA tools you'll need)
phpcompatibility/php-compatibility suggests installing roave/security-advisories (dev-master || Helps prevent installing dependencies with known security issues.)
phpcompatibility/phpcompatibility-paragonie suggests installing roave/security-advisories (dev-master || Helps prevent installing dependencies with known security issues.)
phpcompatibility/phpcompatibility-wp suggests installing roave/security-advisories (dev-master || Helps prevent installing dependencies with known security issues.)
Writing lock file
Generating autoload files
```

2. Run

```sh
vendor/bin/phpcs -i
```
Which gives out:

```sh
The installed coding standards are PEAR, Zend, PSR2, MySource, Squiz, PSR1, PSR12, PHPCompatibility, PHPCompatibilityParagonieRandomCompat, PHPCompatibilityParagonieSodiumCompat, PHPCompatibilityWP, WordPress, WordPress-Extra, WordPress-Docs, WordPress-Core and WPThemeReview
```

Run 

```sh
vendor/bin/phpcs -p . --standard=WPThemeReview
```

Which gives the error

```sh
Uh oh... can't find WPCS.

If you use Composer, please run `composer install`.
Otherwise, make sure you set a `WPCS_DIR` environment variable
pointing to the WPCS directory.
```

When I tried the same with the older version (without specifying a branch), all worked fine.

In the `autoload.php` the 

```php
file_exists( $wpcsDir . $ds . 'WordPress' . $ds . 'PHPCSAliases.php' );
```

Will return `false` which causes the error. Seeing how the Aliases were removed, the entire

```php
// Try and load the WPCS class aliases file.
if ( false !== $wpcsDir && file_exists( $wpcsDir . $ds . 'WordPress' . $ds . 'PHPCSAliases.php' ) ) {
	require_once $wpcsDir . $ds . 'WordPress' . $ds . 'PHPCSAliases.php';
} else {
	echo 'Uh oh... can\'t find WPCS.

If you use Composer, please run `composer install`.
Otherwise, make sure you set a `WPCS_DIR` environment variable
pointing to the WPCS directory.
';

	die( 1 );
}
```

block needs to be removed.

### CODE REVIEW

**Comment by dingo-d** [comment]
@jrfnl Could you look if this is ok, so that I can publish a make post on the theme review make blog? 🙂

**Comment by jrfnl** [comment]
Oh and if it all works as expected once the file is dropped + the autoload command removed from the ruleset, the unit test instructions in the `CONTRIBUTING.MD` file can also be shortened a lot.

**Comment by dingo-d** [comment]
Ok I'll fix this, test and update the PR :+1:

**Comment by dingo-d** [comment]
I have change the contributing documentation, is this ok to merge?

**Comment by jrfnl** [comment]
@dingo-d You have removed a little _too_ much now. The Contributing guidelines should still explain how to run the unit tests with a git clone based setup.

**Comment by dingo-d** [comment]
I've made the changes, not sure if I added the correct parts, can you check if it's ok now? Thanks!

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -91,6 +91,20 @@ composer install
...
 
+### Other setups
+
+If you have PHP_CodeSniffer (PHPCS) and/or WordPress Coding Standards (WordPressCS) already installed, for instance as git clones, because you either contribute to these libraries as well or you want to develop for or test WPThemeReview with bleeding edge versions of either PHP_CodeSniffer or WordPress Coding Standards, you need to take some additional steps to make it all work.
+
+First, make sure you also have PHPCompatibility installed and make sure the `installed_paths` for PHP_CodeSniffer is set up correctly.
+
+You can see how this can be done by reading the official PHPCS [documentation](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Configuration-Options#setting-the-installed-standard-paths).
+
+You can copy the `phpunit.xml.dist` file, rename it to `phpunit.xml` and adjust the bootstrap line to point to where PHPCS is installed.
```
Comment (jrfnl) [inline .github/CONTRIBUTING.md]: Suggestion:

> Next, you need to make sure that PHPUnit can find the PHPCS unit test bootstrap file.
> There are two ways to do this:
> 1. You can copy the `phpunit.xml.dist` file, rename it to `phpunit.xml` and adjust the bootstrap line to point to where PHPCS is installed on your system.
> 2. Alternatively, you can add `--bootstrap="/path/to/PHPCS/tests/bootstrap.php"` to the `phpunit` command when you invoke it on the command line.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -117,7 +131,7 @@ Time: 16.46 seconds, Memory: 48.00MB
 OK (12 tests, 0 assertions)
 ```
 
-If you didn't install PHPCS/WPCS/PHPUnit using Composer, you will need to type the above command in to run the unit tests. Make sure you replace the path to PHPUnit and the path to PHPCS when you do.
+If you didn't install PHPCS/WPCS/PHPUnit using Composer, you will need to type the above command in to run the unit tests. Make sure you replace the path to PHPUnit and the path to PHPCS when you do, and add the `--bootstrap="/path/to/PHPCS/tests/bootstrap.php"` part when running the unit test.
```
Comment (jrfnl) [inline .github/CONTRIBUTING.md]: ... and, _if you didn't setup your own `phpunit.xml` file_, add `--bootstrap="/path/to/PHPCS/tests/bootstrap.php"`.

**Comment by dingo-d** [comment]
Added the fixes 🙂 Thanks for all the help @jrfnl

**Comment by dingo-d** [comment]
Go ahead and squash merge, better not to risk me messing something up 😄

**Pull request merged**